src/Umbraco.Web/Editors/BackOfficeController.cs

```diff
@@ -341,6 +341,7 @@ public JavaScriptResult ServerVariables()
                                 string.Join(",", UmbracoConfig.For.UmbracoSettings().Content.ImageFileTypes)
                             },
                             {"keepUserLoggedIn", UmbracoConfig.For.UmbracoSettings().Security.KeepUserLoggedIn},
+                            {"cssPath", IOHelper.ResolveUrl(SystemDirectories.Css).TrimEnd('/')},

```

Comment:
As with other 'paths' that are part of the server variables, this should exist under: `Umbraco.Sys.ServerVariables.umbracoSettings.umbracoSettings.cssPath`